src/module/guards/logged-in.guard.ts

```diff
@@ -0,0 +1,34 @@
+import { Inject, Injectable } from '@angular/core'
+import { CanActivate, Router } from '@angular/router'
+import { map } from 'rxjs/operators'
+import { Observable } from 'rxjs'
+
+import {
+  AuthProcessService,
+  NgxAuthFirebaseUIConfig,
+  NgxAuthFirebaseUIConfigToken,
+} from '../ngx-auth-firebase-u-i.module'
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LoggedInGuard implements CanActivate {
```

Comment:
What do you think about the name of the guard ... `AuthGuard` vs `LoggedInGuard` ?